tests/format/scss/quotes/quotes.scss

```diff
@@ -0,0 +1,2 @@
+@use "module";
+@forward "module";
```

Comment:
We need more tests here, because we use parser here and it can break output, please add:
```
@use 'library' with (
  $black: #222,
  $border-radius: 0.1rem
);
```

ideally even more